### PR TITLE
Update Legacy AutoLogoutModal Theme

### PR DESCRIPTION
**Pull request by Jon-edge**
iOS:
image

Android:
image

### CHANGELOG

- changed: Update UI theme for Settings AutoLogoutModal

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [X] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1200841872621585

### CODE REVIEW

**Comment by swansontec** [comment]
After merging this, the following files are unused:

src/components/legacy/AirshipModal.tsx
src/components/legacy/Buttons/SecondaryButton.ui.tsx
src/components/legacy/Buttons/TertiaryButton.ui.tsx
src/components/legacy/ContentArea.tsx
src/components/legacy/IconCircle.tsx
src/components/legacy/LadderLayout.tsx

Congratulations! You get the honor of deleting them, after all these years of sticking around, gooping up our codebase.

File: src/components/modals/AutoLogoutModal.tsx [hunk 1]
```diff
@@ -1,108 +1,93 @@
...
   bridge: AirshipBridge<number | undefined>
   autoLogoutTimeInSeconds: number
 }
 
-type State = DisplayTime
+export const AutoLogoutModal = (props: Props) => {
+  const { autoLogoutTimeInSeconds, bridge } = props
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  const textColor = theme.pickerTextDark
```
Comment (swansontec) [inline src/components/modals/AutoLogoutModal.tsx:23]: Would it make sense to delete `pickerTextDark` and `pickerTextLight` from the theme, and then just use normal `theme.primaryText` for this? Or, at the very least, just merge the two into a single `pickerText` entry?

**Review by swansontec — CHANGES_REQUESTED** [review]

**Comment by Jon-edge** [comment]
Can't really change much about the picker for Android. We could probably benefit from a common component instead of trying to massage the native UI further....

**Review by swansontec — APPROVED** [review]
This is great now!

**Pull request merged**